default.nix

```diff
@@ -1,16 +1,3 @@
-{ mkDerivation, reflex, reflex-dom, file-embed
-}:
-
-mkDerivation {
-  pname = "reflex-todomvc";
-  version = "0.1";
-  src = builtins.filterSource (path: type: baseNameOf path != ".git") ./.;
-  isExecutable = true;
-  isLibrary = true;
-  buildDepends = [
-    reflex
-    reflex-dom
-    file-embed
-  ];
-  license = null;
-}
+{ nixpkgs ? import <nixpkgs> {}, compiler ? "ghcjs" }:
+	    nixpkgs.pkgs.haskell.packages.${compiler}.callPackage ./reflex-todomvc.nix
+	   { }
```

Comment:
@ryantrinkle after updating `github.json` in `reflex-todomvc`
```json
{
  "owner": "dmjio",
  "repo": "reflex-todomvc",
  "rev": "4dc7079c35d7d8062643703bec9369585f64bee5", -- github hash
  "sha256": "18yfvrppbkq45zs6w5a5j9iyqdbskmx95zysvqv5gqjd7fvjmbl2"
}
```
It seems to have worked for me... what settings are you passing into `nix-hash` ? Is it simple `nix-hash --type sha256` ?